packages/react-core/src/components/Nav/examples/Nav.md

```diff
@@ -48,9 +48,9 @@ The following example shows two navigation groups, each with a `title` prop pass
 
 ```
 
-### Legacy tertiary
+### Deprecated tertiary
```

Comment:
In the Nave demos @mcarrano suggested putting the deprecated at the end of the title in brackets.  So "Legacy tertiary nav" became Tertiary nav (deprecated).  Can we do the same here?

"Tertiary (deprecated)"